script/compile

```diff
@@ -59,13 +66,21 @@ args=( "-jar" "$JAR"
        "-H:Log=registerResource:"
        "-H:EnableURLProtocols=http,https"
        "--enable-all-security-services"
+       "--allow-incomplete-classpath"
        "-H:+JNI"
        "--verbose"
        "--no-fallback"
        "--no-server"
        "--report-unsupported-elements-at-runtime"
-       "-H:IncludeResources=org/hsqldb/.*\.properties" "-H:IncludeResources=org/hsqldb/.*\.sql"
+       "-H:+AddAllCharsets"
```

Comment:
This was needed so to support mssql's [collation](https://docs.microsoft.com/en-us/sql/relational-databases/collations/collation-and-unicode-support?view=sql-server-ver15).